algorithms/common/buffer/priortized_replay_buffer.py

```diff
@@ -59,27 +71,36 @@ def __init__(self, buffer_size: int, batch_size: int, alpha: float = 0.6):
         self.min_tree = MinSegmentTree(tree_capacity)
         self._max_priority = 1.0
 
+        # for init priority of demo
+        self.tree_idx = self.demo_size
+        for i in range(self.demo_size):
+            self.sum_tree[i] = self._max_priority ** self.alpha
+            self.min_tree[i] = self._max_priority ** self.alpha
+
     def add(
         self,
-        state: np.ndarray,
-        action: np.ndarray,
-        reward: np.float64,
-        next_state: np.ndarray,
-        done: bool,
-    ):
+        transition: Tuple[np.ndarray, Union[int, np.ndarray], float, np.ndarray, bool],
```

Comment:
The type of action would be one thing rather than Union.

Reply:
The types of action vary depending on the environment. For example, PongDeterministic-v4 has action of int type. However, In the Pendulum-v0, the action type is np.ndarray of float32.

Reply:
I mean when data is added buffer, already changes the type on agent class.

Reply:
I fixed it. But demo's action is not yet np.ndarray type. We have to make or get new demo data.